apps/site-app/src/containers/AddAccount/Generate/Intro/index.tsx

```diff
@@ -61,6 +88,31 @@ export const Intro = () => {
         <FormattedMessage {...buttonGo} />
       </ButtonPrimary>
       <Decoration />
+
+
+      <ModalOperation
```

Comment:
Did not bother more than that for this as it is very temporary and should be real help doc soon.